zgw_consumers/models/services.py

```diff
@@ -16,11 +16,14 @@
 
 from zgw_consumers import settings as zgw_settings
 
-from ..client import ZGWClient, get_client_class
 from ..constants import APITypes, AuthTypes, NLXDirectories
 from ..query import ServiceManager
 from .abstract import RestAPIService
 
+if TYPE_CHECKING:
+    # Avoid circular import
+    from ..client import ZGWClient
```

Comment:
this is odd - it worked before moving this to the legacy module, why does it break now?

Reply:
The new stuff in `zgw_consumers.client` is itself importing `Service`, but that might not be necessary if I go with https://github.com/maykinmedia/zgw-consumers/pull/73#discussion_r1350292086